HW0/P5/P5.txt

```diff
@@ -1 +1,23 @@
-# Your answers here.
+# Qing Zhao (HUID: 80985053)’s answer for P5
+
+1. 255 seconds.
+
+2. The time for each of them to count 256 / 8 = 32 bags is 31 s; summing up 8 numbers takes at least 3s (2 ^ 3 = 8). Therefore, 8 cashiers can do this in 34 seconds.
+
+3. 8 seconds.
+
+4. See ‘P5.png’
+
+5. With infinite employees, it will take 8 + 7 = 15 seconds to count 256 bags;
+
+6. 1 bag each: 256 + log2(256) + log2(256) = 272 seconds;
```

Comment:
Are you assuming 256 employees? If that's the case then it should be (256/256)-1+2*log2(256) = 16 seconds